lib/sequella/service.rb

```diff
@@ -7,8 +7,11 @@ class << self
       # Start the Sequel connection with the configured database
       def start(config)
         params = config.to_hash.select { |k, v| !v.nil? }
+        # using sequel default values.
+        max_connections = params[:max_connections]||4
+        pool_timeout = params[:pool_timeout]||5
 
-        @@connection = establish_connection connection_string(params)
+        @@connection = establish_connection(connection_string(params),max_connections,pool_timeout)
```

Comment:
Also, it occurs to me that it might be better to pass this as a hash to `#establish_connection`, in case we get more params in the future.  The calling code would then look something like

```
@@connection = establish_connection connection_string(params), max_connections: max_connections, pool_timeout: pool_timeout